.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "prettier.requireConfig": true
```

Comment:
if you would like to add this file to repo - could you please add it in a separate  PR with description - what it does and why we need it?

Reply:
Removed